modules/hosted-control-planes-support-matrix.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/index.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hosted-control-planes-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Note the following scope of compatibility and support for hosted control planes. The management cluster is based on the installer-provisioned infrastructure (IPI).
+
+.Compatibility and support matrix
```

Comment:
Can we make it clear which MCE versions are we upgrading from and to? Also would be great if we can add or reference OCP versions as well. cc @LiangquanLi930

Reply:
It's not immediately obvious how to read the table.